processing/src/main/java/org/apache/druid/segment/nested/DictionaryIdLookup.java

```diff
@@ -135,7 +135,9 @@ public int lookupString(@Nullable String value)
   public int lookupLong(@Nullable Long value)
   {
     if (longDictionary == null) {
-      Path longFile = makeTempFile(name + NestedCommonFormatColumnSerializer.LONG_DICTIONARY_FILE_NAME);
+      final Path longFile = makeTempFile(
```

Comment:
note: seems like `makeTempFile` is a class method; it could be possible to push the `StringUtils.urlEncode(name)` into that; so that call sites could look like:
```
makeTempFile(NestedCommonFormatColumnSerializer.LONG_DICTIONARY_FILE_NAME)
```
the method already appends `.tmp` ; so it could pick up this as well

Reply:
I was thinking of that too but if we are doing that, we should change the name of the method so that encoding is not done twice?